Hitchhikers_Guide_to_the_Datalake.md

```diff
@@ -194,6 +194,8 @@ The table below provides a quick overview of how ACLs and RBACs can be used to m
 | **Limits** |	2000 RBACs in a subscription |	32 ACLs (effectively 28 ACLs) per file, 32 ACLs (effectively 28 ACLs) per folder, default and access ACLs each |
 | **Supported levels of permission** |	[Built-in RBACs](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth-aad-rbac-portal#rbac-roles-for-blobs-and-queues) or [custom RBACs](https://docs.microsoft.com/en-us/azure/role-based-access-control/custom-roles) | [ACL permissions](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-access-control#levels-of-permission) |
 
+When using RBAC at the file system level as the only mechanism for data access control, be cautious of the 2000 limit, particularly if you are likely to have a large number of file systems. You can view the number of role assigments per subscription in any of the access control (IAM) blades in the portal.
```

Comment:
Can you please add a relevant link to the portal or docs? That might be a helpful reference.

Reply:
Done